csaf/providermetaloader.go

```diff
@@ -108,8 +109,51 @@ func NewProviderMetadataLoader(client util.Client) *ProviderMetadataLoader {
 	}
 }
 
-// Load loads a provider metadata for a given path.
-// If the domain starts with `https://` it only attemps to load
```

Comment:
I believe the comment for documentation "If the domain starts with `https://` it only attemps to load this as PMD" or so, should stay in.